includes/Admin/Notices/Helper.php

```diff
@@ -18,13 +18,14 @@ class Helper {
      *
      * @return array | void
      */
-    public static function dokan_get_admin_notices() {
+    public static function dokan_get_admin_notices( $notice_scope = 'local' ) {
```

Comment:
Pls do not modify this files. Undo all changes.

Reply:
vai, done.